src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```
@@ -84,7 +85,7 @@ object TSVFormatter {
     * @return IndexedSeq of header Strings
     */
   def makeMembershipHeaders(entityType: String): IndexedSeq[String] = {
-    IndexedSeq[String](s"${TsvTypes.MEMBERSHIP}:${entityType}_id", memberTypeFromEntityType(entityType))
+    IndexedSeq[String](s"${TsvTypes.MEMBERSHIP}:${entityType}_id", FlexibleModelSchema.memberTypeFromEntityType(entityType).getOrElse(""))
```

I think we want just `.get` instead of `.getOrElse`. If getting the membership type fails, we want to raise that exception, not use `""` as a header.